mirage/psf/psf_selection.py

```diff
@@ -351,7 +356,8 @@ def get_library_file(instrument, detector, filt, pupil, wfe, wfe_group,
             # If so, add to the list of all matches
             if match:
                 matches.append(filename)
-        except KeyError:
+        except KeyError as e:
+            print(e)
```

Comment:
Is just a print sufficient for this error? Would it be better to use warnings.warn() instead?

Reply:
That's a good idea, since this will catch problems with files that may not be used during a given run of Mirage.